pkgs/desktops/pantheon/apps/switchboard-plugs/about/fix-translations.patch

```diff
@@ -0,0 +1,56 @@
+From 0be5886613e3de5cc1c4a5cf48ff2a95a5c081a3 Mon Sep 17 00:00:00 2001
+From: Bobby Rong <rjl931189261@126.com>
+Date: Mon, 9 Aug 2021 18:44:46 +0800
+Subject: [PATCH] Fix translations
+
+---
+ src/Config.vala.in | 5 ++---
+ src/Plug.vala      | 5 +++++
+ src/meson.build    | 3 ++-
+ 3 files changed, 9 insertions(+), 4 deletions(-)
+
+diff --git a/src/Config.vala.in b/src/Config.vala.in
+index 6df4206..13a45d2 100644
+--- a/src/Config.vala.in
++++ b/src/Config.vala.in
+@@ -17,6 +17,5 @@
+  *  Boston, MA 02110-1301 USA.
+  */
+ 
+-namespace About {
+-    private const string GETTEXT_PACKAGE = "@GETTEXT_PACKAGE@";
+-}
++public const string GETTEXT_PACKAGE = "@GETTEXT_PACKAGE@";
++public const string LOCALEDIR = @LOCALEDIR@;
+diff --git a/src/Plug.vala b/src/Plug.vala
+index dbc20d4..c665f54 100644
+--- a/src/Plug.vala
++++ b/src/Plug.vala
+@@ -120,6 +120,11 @@ public class About.Plug : Switchboard.Plug {
+ 
+ public Switchboard.Plug get_plug (Module module) {
+     debug ("Activating System plug");
++    debug (LOCALEDIR);
++    Intl.setlocale (LocaleCategory.ALL, "");
++    GLib.Intl.bindtextdomain (GETTEXT_PACKAGE, LOCALEDIR);
++    GLib.Intl.bind_textdomain_codeset (GETTEXT_PACKAGE, "UTF-8");
++    GLib.Intl.textdomain (GETTEXT_PACKAGE);
+     var plug = new About.Plug ();
+     return plug;
+ }
+diff --git a/src/meson.build b/src/meson.build
+index 5e8d30e..cdc1c32 100644
+--- a/src/meson.build
++++ b/src/meson.build
+@@ -14,7 +14,8 @@ switchboard_dep = dependency('switchboard-2.0')
+ switchboard_plugsdir = switchboard_dep.get_pkgconfig_variable('plugsdir', define_variable: ['libdir', libdir])
+ 
+ config_data = configuration_data()
+-config_data.set('GETTEXT_PACKAGE', meson.project_name())
++config_data.set('GETTEXT_PACKAGE', meson.project_name() + '-plug')
```

Comment:
Just a quick note, I wasted 20 minutes on this diff, maybe report to upstream later. 🙃